src/extension.ts

```diff
@@ -296,6 +296,7 @@ export async function activate(context: ExtensionContext): Promise<void> {
         lightSpeedManager,
         pythonInterpreterManager,
       );
+      await lightSpeedManager.lightspeedExplorerProvider.refreshWebView();
```

Comment:
This was needed for the case that a playbook was not initially recognized as an Ansible file and the user manually set the Ansible language code.  The webview needs to be refreshed at that timing.

Reply:
Do we actually need to `await` here?

Reply:
Maybe not.  Let me try w/o await.

Reply:
It worked without `await`. I'll get rid of that.